src/compo/viirs_l1bnc2ioda.py

```diff
@@ -36,8 +38,12 @@
 # VIIRS M-band 11 reflective channels central wavelength. Do not change list below.
 wavelength = [0.412, 0.445, 0.488, 0.555, 0.672, 0.746,
               0.865, 1.24, 1.378, 1.61, 2.25]
+orbit_height = 829.       # in km
+earth_radius = 6378.137   # in km, mean earth radius
 speed_light = 2.99792458E8
 frequency = speed_light*1.0E6/np.array(wavelength)
+deg2rad = np.pi/180.
```

Comment:
Couldn't you just use pi in a few places you need this?

Reply:
Yes, I can update it.